### PR TITLE
Cif 315 improve consistency and standardization of cif code for layers

### PR DESCRIPTION
**Pull request by kcartier-wri**
## How was this patch tested?
Generated layer files with prior and updated code and compared. Results were identical as measured with Meld.

### CODE REVIEW

File: city_metrix/layers/albedo.py [hunk 1]
```diff
@@ -34,19 +32,33 @@ def get_data(self, bbox):
...
         def mask_clouds_and_rescale(im):
-            clouds = ee.Image(im.get('cloud_mask')).select('probability')
-            return im.updateMask(clouds.lt(MAX_CLOUD_PROB)).divide(10000)
+            clouds = ee.Image(im.get('cloud_mask')
+                              ).select('probability')
+            mask = im.updateMask(clouds
+                                 .lt(MAX_CLOUD_PROB)
+                                 ).divide(10000)
+
+            return mask
```
Comment (chrowe) [inline city_metrix/layers/albedo.py:61]: Should this match the others?

File: city_metrix/layers/albedo.py [hunk 2]
```diff
@@ -89,20 +107,25 @@ def calc_s2_albedo(image):
...
-        data = (get_image_collection(
-            ee.ImageCollection(albedo_mean), bbox, self.spatial_resolution, "albedo")
-                .albedo_mean)
+        albedo_mean_ic = ee.ImageCollection(albedo_mean)
+        data = get_image_collection(
+            albedo_mean_ic,
+            bbox,
+            self.spatial_resolution,
+            "albedo"
+        ).albedo_mean
```
Comment (chrowe) [inline city_metrix/layers/albedo.py:126]: Do we want the ) on the line above in cases like this?

**Review by chrowe — CHANGES_REQUESTED** [review]
Looks good to me other than the comments I made.

**Comment by kcartier-wri** [comment]
I've decided that I don't want to go further down the rabbit hole on the formatting topic.

My main criteria with these changes were to:
1. make the code more easily comprehensible by expanding functional code that was previously specified in one line into multiple lines with expressive indentation. We shouldn't obsess over the specifics - the code should just be easily readable, even with some stylistic inconsistency.
2. remove complex return statements
3. eliminate unused imports
4. use meaningful variable names
5. etc.

I've already tested the code in this PR so am reluctant to make further re-formatting changes,

File: city_metrix/layers/albedo.py (hunk 2 above)
Comment (kcartier-wri) [reply city_metrix/layers/albedo.py:126]: I think it's comprehensible as is.

**Review by jterry64 — APPROVED** [review]
This looks good to me, but just curious, did you change all this manually? One thing I never set up here we have in most other repos is pre-commit, which can run a bunch of automatic linters on commit that'll do most of this for you. 

Here's an example of a pre-commit config we have with linters like isort, black, and flake8 that'll fix spacing, ensure you're following PEP, etc.: https://github.com/wri/gfw-data-api/blob/master/.pre-commit-config.yaml

Your changes mostly look correct, but slightly different than the formatting I've seen in other repos.

**Comment by kcartier-wri** [comment]
@jterry64 I strongly agree that an automated, consistent reformatting would be best. I tried reformatting with VS Black and the default formatter in Pycharm but they didn't seem to do much. I'll follow-up with your suggestion above. Thanks

**Review by weiqi-tori — APPROVED** [review]

**Review by chrowe — APPROVED** [review]

**Pull request merged**